style1.css

```diff
@@ -0,0 +1,231 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  background-color: transparent;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+html {
+  width: 375px;
+}
+
+.container li,
+.myself li {
+  list-style: none;
+  padding-top: 10px;
+  padding-right: 12px;
+}
+
+.container {
+  background-image: url(images/headershapesmobile1.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 584px;
+}
+
+.color {
+  color: #6070ff;
+}
+
+.container button {
+  background-color: transparent;
+  border: none;
+  outline: none;
+}
+
+/* HEADER  */
+.header {
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you remove the duplicate files `index1.html` and `style1.css` as they have the same code as the original files  `index.html` and `style.css` to make your repository cleaner.